quicklens/src/test/scala/com/softwaremill/quicklens/RepeatedModifyAllTest.scala

```diff
@@ -0,0 +1,276 @@
+package com.softwaremill.quicklens
+
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+class RepeatedModifyAllTest extends AnyFlatSpec with Matchers {
+  import RepeatedModifyAllTest._
+
+  it should "expand a very long repeated function" in {
+    val c6 = C6(1)
+    val c5 = C5(c6)
+    val c4 = C4(c5)
+    val c3 = C3(c4)
+    val c2 = C2(c3)
+    val c1 = C1(c2)
+
+    val c6e = C6(2)
+    val c5e = C5(c6e)
+    val c4e = C4(c5e)
+    val c3e = C3(c4e)
+    val c2e = C2(c3e)
+    val c1e = C1(c2e)
+
+    val res = c1
+      .modifyAll(
+        _.a.a.a.a.a.a,
```

Comment:
One problem with this test is I'm not sure what would it mean to set the same field path several times. Unless this is only meant to test the generated bytecode size, not the functionality.